integration/simulation/simulation.go

```diff
@@ -44,8 +44,8 @@ type Simulation struct {
 	SimulationTime   time.Duration
 	Stats            *stats.Stats
 	Params           *params.SimParams
-	LogChannels      map[string]chan common.IDAndLog // Maps an owner to the channel on which they receive logs for all their wallets.
-	Subscriptions    []ethereum.Subscription         // A slice of all created event subscriptions.
+	LogChannels      map[string][]chan common.IDAndLog // Maps an owner to the channels on which they receive logs for each client.
```

Comment:
We need a slice of channels now, one for each client. Otherwise, with every client subscribing, we'd see dupes (one per client), which is what we're testing for.